composer.json

```diff
@@ -74,6 +74,7 @@
         "phpseclib/mcrypt_compat": "1.0.8",
         "phpseclib/phpseclib": "2.0.*",
         "ramsey/uuid": "~3.8.0",
+        "s1lentium/iptools": "^1.1",
```

Comment:
Seems like should be also added to the framework and setup composer.json files

Reply:
I've added this requirement to `lib/internal/Magento/Framework/composer.json`, but didn't see any `composer.json` files within `setup/`.